test/test.js

```diff
@@ -96,6 +97,25 @@ test('plan assertions with support for promises', function (t) {
 	});
 });
 
```

Comment:
Might we worth extracting all the Observable related test into a separate test file, since there are so many now.